lib/Engine_KarplusRings.sc

```diff
@@ -0,0 +1,74 @@
+// Pluck uGen workout. Shout me @burn on llllllll.co if you need any assistance.
+
+Engine_KarplusRings : CroneEngine {
+	var pg;
+	var amp=0.1;
+	var freq = 440;
+	var decay = 5;
+	var coef = 0.1;
+  var lpf_freq = 3000;
```

Comment:
Is there an issue with the indent on this line? (I don't know enough about SC to know if that matters!)

Reply:
oh! no issue running on a fresh install, but fixed the indent for optics :)
thank you for spotting!!